modules/core/src/main/scala/gem/Step.scala

```diff
@@ -142,13 +144,13 @@ object GcalStep {
 // Science
 // ----------------------------------------------------------------------------
 
-final case class ScienceStep[I](instrument: I, telescope: Telescope) extends Step[I] {
+final case class ScienceStep[I](instrument: I, telescope: TelescopeConfig) extends Step[I] {
   def stepType = Step.Science
 }
 
 object ScienceStep {
```

Comment:
This comes from the sequence model work I shared with everyone last year.  It seems like we'll want to be working with just one sequence model in the end, but it also feels like the program model work is still a bit preliminary to be relied upon in the seqexec.  I'll want to have the freedom to discover what works best and change things around for a while and I imagine the seqexec team will too.  So I'm not sure whether it makes sense to try to develop them together at this point.  I'm hoping we can refactor as necessary to put them together if necessary.